scripts/expert/Botania.zs

```diff
@@ -65,16 +65,16 @@ mods.thaumcraft.Infusion.registerRecipe("GaiaPylon", "", <botania:pylon:2>, 6, [
 
 # -Catalysts
 recipes.remove(<botania:alchemycatalyst>);
-recipes.addShaped(<botania:alchemycatalyst>, [[<ore:livingrock>, <bewitchment:focal_chalk>, <ore:livingrock>], [<bloodmagic:slate:1>, <thaumcraft:salis_mundus>, <bloodmagic:slate:1>], [<ore:livingrock>, <ore:blockblaze>, <ore:livingrock>]]);
```

Comment:
Typo in OreDict - it should be <ore:blockBlaze> with lowercase "b" - no need to change the Livingrock OreDicts.